packages/flutter/lib/src/painting/box_decoration.dart

```diff
@@ -126,14 +126,14 @@ class BoxDecoration extends Decoration {
 
   @override
   bool debugAssertIsValid() {
-    assert(shape != BoxShape.circle || borderRadius == null); // Can't have a border radius if you're a circle.
+    assert(shape == BoxShape.rectangle || borderRadius == null); // Can't have a border radius if you're a circle or oval.
```

Comment:
```suggestion
    assert(shape == BoxShape.rectangle || borderRadius == null); // Can't have a border radius if you're a circle or an oval.
```

Reply:
Fixed, but this is definitely not following the 80-column limit.
<img width="640" alt="image" src="https://user-images.githubusercontent.com/351125/178824485-70dd1a4d-0968-4d64-91df-ceae5ac3a4fc.png">